.github/workflows/check-catalyst.yaml

```diff
@@ -129,9 +129,11 @@ jobs:
     - name: Clone LLVM Submodule
       if: |
         steps.cache-llvm-source.outputs.cache-hit != 'true'
-      run: |
-        git submodule update --init mlir/llvm-project
-        [ $(cd mlir/llvm-project && git rev-parse HEAD) == ${{ steps.llvm-hash.outputs.llvm-hash }} ]
+      uses: actions/checkout@v3
+      with:
+        repository: llvm/llvm-project
+        ref: ${{ steps.llvm-hash.outputs.llvm-hash }}
+        path: mlir/llvm-project
```

Comment:
Could these changes also be added to the wheel scripts?

Reply:
Oh good idea!

Reply:
https://github.com/PennyLaneAI/catalyst/pull/45/commits/e98b40e9fd9b64428239d1d8943fa193a20b7b1b